manifests/zabbixapi.pp

```diff
@@ -0,0 +1,33 @@
+# @summary This will install the zabbixapi gem.
+#
+# @param zabbix_version This is the zabbix version.
+# @param puppetgem Provider for the zabbixapi gem package.
+class zabbix::zabbixapi (
+  $zabbix_version = $zabbix::params::zabbix_version,
+  $puppetgem = $zabbix::params::puppetgem,
+) inherits zabbix::params {
+  # Determine correct zabbixapi version.
+  case $zabbix_version {
+    /^[56]\.[024]/: {
+      $zabbixapi_version = '5.0.0-alpha1'
```

Comment:
Note to self: https://rubygems.org/gems/zabbixapi does only list a 5.0.0-alpha1 from 2021.